e2e/suites/convert-entity-template.test.ts

```diff
@@ -0,0 +1,62 @@
+/*global page*/
+
+import { ElementHandle } from 'puppeteer';
+import { createEntity } from '../helpers/createEntity';
+import { createTemplate } from '../helpers/createTemplate';
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+import insertFixtures from '../helpers/insertFixtures';
+import disableTransitions from '../helpers/disableTransitions';
+
+const setupPreFlights = async (): Promise<void> => {
+  await insertFixtures();
+  await proxyMock();
+  await adminLogin();
+};
+
+const setupTest = async () => {
+  await createTemplate('Without image');
+  await createTemplate('With image');
+  await createEntity('Without image', {
+    pdf: `${__dirname}/test_files/valid.pdf`,
+    supportingFile: `${__dirname}/test_files/batman.jpg`,
+  });
+};
+
+describe('Convert entity template', () => {
```

Comment:
I don't think this is testing "Convert entity template" but actually testing that the image is being rendered. We need to find a better description and file name.